intel_extension_for_transformers/llm/runtime/graph/models/baichuan/baichuan.h

```diff
@@ -0,0 +1,51 @@
+//  Copyright (c) 2023 Intel Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#ifndef BAICHUAN_H
+#define BAICHUAN_H
+
+#include "models/model_utils/model_files.h"
+#include "models/model_utils/model_types.h"
+
+enum baichuan_model {
+  BAICHUAN_UNKNOWN,
+  BAICHUAN_13B,
+};
+
+static const model_scratch baichuan_mem_req(int n_layers) {
+  switch (n_layers) {
+    case 40:
+      return {8192ull * MB, 8192ull * MB, 8192ull * MB};
+    // TODO(hengyu): add more variants besides 6B
```

Comment:
?

Reply:
All head files of model have this comment and mention you.

Reply:
quite sad to be notified of this...

Reply:
let me remove these commnets in this PR.

Reply:
Fixed. https://github.com/intel/intel-extension-for-transformers/pull/376/commits/f229a212e552141ddea739ed47ef6b7133931e68